content/en/continuous_integration/pipelines/azure.md

```diff
@@ -34,6 +34,31 @@ Set up tracing on Azure Pipelines to gain real time insights into your CI/CD wor
 | [Execution time][20] | Execution time  | View the amount of time pipelines have been running jobs. |
 
 ## Configure the Datadog integration
+{{< tabs >}} {{% tab "Datadog Integration (recommended)" %}}
+
+### Enable CI Visibility in Datadog
```

Comment:
This heading seems unnecessary because we already have a heading and a tab title right above it, and the sentence below also says "enable CI Visibility."
```suggestion
```